Beginners_guide/README.rst

```diff
@@ -3,32 +3,14 @@ Beginner's guide
 
 The Beginners Guide contains introductory notebooks aimed at introducing Jupyter Notebooks, Digital Earth Australia (DEA), and how to load, plot and interact with data from DEA. These notebooks are designed to be worked through in the following order:
 
-.. toctree::
```

Comment:
We used to use these readme tables of contents to control what notebooks are included in the docs (we have some notebooks that we sync to the Sandbox but don't render in the docs). Now that they've been removed, is there another place where we can do this, e.g. view and modify the notebooks that will be rendered in Knowledge Hub?

Reply:
Hi Robbi, I will be keeping all of these `toctree` elements within this Notebooks repo except this one. That is because I am moving control of the top-level toctree to the dea-docs repo, as this allows more flexibility in how the sidebar displays on the DEA Knowledge Hub. However, I will explain it better when I make a new PR.